censustree/censustree.go

```diff
@@ -34,7 +34,7 @@ const nLevels = 256
 // New returns a new Tree, if there already is a Tree in the
 // database, it will load it.
 func New(wTx db.WriteTx, opts Options) (
-	*Tree, error) {
+	_ *Tree, err error) {
```

Comment:
probably undo these changes now :) also no need to split these func decls into two lines, they're fairly short